features/step_definitions/run_steps.rb

```diff
@@ -59,14 +59,9 @@
 
 
 Then(/^it runs the Git commands$/) do |expected_commands|
-  sha_regex = /\[SHA:(.+?)\]/
-
   # Replace SHA placeholders with the real SHAs
```

Comment:
Since ERB is slow, I want to keep the comment here to indicate why we have to do it here. Updated it though.